extensions/wikia/FacebookClient/scripts/FacebookClient.facebookTags.js

```diff
@@ -12,7 +12,9 @@ mw.hook('wikipage.content').add(function ($content) {
 
 	// Checks for XFBML tags or Facebook Page Plugin tags
 	function facebookTagsOnPage() {
-		var numOfFacebookTags = $content.find('[data-type="xfbml-tag"], .fb-page').length;
+        // All fb social plugins have a class which starts with 'fb-'
```

Comment:
I would consider to move it one indentation to the right, so the comment matches the line of code it's describing :)

Also, thanks for adding this comment as it's giving a lot of clarity to the complicated selector.